proto/process/agent.proto

```diff
@@ -641,7 +636,7 @@ enum EphemeralPortState {
 message Connection {
 	reserved 2, 3, 4, 7, 8, 9, 13, 14, 15, 35;
 	// Please update when adding fields
-	// next identifier:  46
+	// next identifier:  44
```

Comment:
Perhaps add another PR after this to bump this back to 46 in-case there are other changes introduced before this gets re-evaluated/merged?

Reply:
We don't have to use the same identifer when we eventually re-add this, since nothing consumes this data yet.